.github/workflows/build-graalvm.yml

```diff
@@ -33,17 +33,17 @@ jobs:
           file: ./target/teamscale-upload
       - name: 'Upload Artifact'
         if: ${{ always() }}
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
For the reviewer: This fixed some issues where the GitHub action would not run due to deprecation of v2